pom.xml

```diff
@@ -99,6 +106,25 @@
             <artifactId>api-security-java</artifactId>
             <version>${api-security-java.version}</version>
         </dependency>
+        <dependency>
```

Comment:
i believe we will also need to update the import statements where SLF4J is currently used to use the companies house package instead

Reply:
Shouldn't be needed? We're still using SLF4j so the code shouldn't need to be changed, this is just changing the underlying logger binding from logback to structured-logging.

Attached screenshots to show human readable logs active in local environment:

<img width="1521" alt="Screenshot 2020-09-02 at 11 14 08" src="https://user-images.githubusercontent.com/10167124/91969303-c8f3cc80-ed0d-11ea-949c-706da0bb2ca6.png">

Reply:
Hmmm looking at another CH service that uses the structured logger, here's an example import of there's https://github.com/companieshouse/efs-submission-api/blob/master/src/main/java/uk/gov/companieshouse/efs/api/interceptor/UserAuthenticationInterceptor.java#L9